modules/gke/main.tf

```diff
@@ -121,3 +121,121 @@ resource "google_container_node_pool" "default" {
     create_before_destroy = true
   }
 }
+
+resource "google_container_node_pool" "app_node_pool" {
+  count      = var.enable_app_node_pool ? 1 : 0
```

Comment:
I think the app_node_pool is not required. We can set sizes and machine types on the default node pool. That should be ok. 

can you cofnirm this is basically a copy of the previous one?

Reply:
confirming I basically cloned the previous one, I thought perhaps we wanted 2 *new* node pools so not to disturb the original and then we'd come back and delete it subsequently, but I will adjust my changes to just modify the machine type variable for the default one and get rid of this extra one, thanks

Reply:
removed now